src/styles/index.scss

```diff
@@ -1,4 +1,5 @@
 @import './reset.scss';
+@import './commons.scss';
```

Comment:
Should go after colors and typography.

`general.scss` should not exist.

Reply:
fixed,
but please note, we can not include sass variables in common.scss cause it will lead to css duplication.
both @import and @use operators combines referenced file css with our file css:
https://sass-lang.com/documentation/at-rules/import/
https://sass-lang.com/documentation/at-rules/use/

so, in order to avoid duplication and address your comment, I renamed general.scss to vars.scss